pkg/deploy/stack/populate/podcontrollers/statefulset.go

```diff
@@ -30,6 +32,9 @@ func statefulSet(stack *input.Stack, service *v1beta1.Service, cp *controllerPar
 			Template: cp.PodTemplateSpec,
 		},
 	}
+	statefulSet.Labels[useTemplatesLabel] = "true"
```

Comment:
What is this label used for?  You have to be very careful with labels you add to the selector because they can't be modified after the fact.  So I try to just use as few as possible in the selector

Reply:
I'm thinking this can just be removed if you use the `rio.catle.io/volume-template` label in the conroller.